ansible/roles/sshkey-host-access/README.md

```diff
@@ -1,32 +1,31 @@
 sshkey-host-access
 =========
 
-Create and move keypairs to new instances for access through Web Shells.
+Enable ssh access from an external host to the playbook target as user
+
+If the host has an existing public key, it can specify a path
+EXTERNAL_HOST_PUBLIC_KEY_PATH (located on the EXTERNAL_HOST), otherwise a
+keypair will be created.
 
 Role Variables
 --------------
 
-| Variable                | Required | Default | Choices                   | Comments                                 |
-|-------------------------|----------|---------|---------------------------|------------------------------------------|
-| ATMOUSERNAME            | yes      |         |                           | Username of user getting the keys        |
-| LOCAL_PUBKEY_DIR        | yes      | `/tmp/` |                           | the directory to transfer the public key locally |
-| KEY_LOC_PATH            | yes      |`/var/lib/gateone/users/{{ ATMOUSERNAME }}/.ssh`|| the directory to transfer the public key |
-| KEY_LOC_OWNER           | yes      | root    | "root", "tomcat7"         | owner of new keys                        |
-| KEY_LOC_GROUP           | yes      | root    | "root", "tomcat7"         | group of new keys                        |
-| KEY_LOC_PREFIX          | yes      | ""      | "_guac", ""               | prefix to name key for specific server   |
-| target_host             | yes      | shell   | "shell", "guac_server"    | hostname for delegated tasks             |
-| FULL_PUBKEY_PATH        | yes      |         |                           | simplifies long string of other variables|
-| FULL_PRIVKEY_PATH       | yes      |         |                           | simplifies long string of other variables|
-| LOCAL_FULL_PUBKEY_PATH  | yes      |         |                           | simplifies long string of other variables|
+| Variable                      | Required   | Default    | Choices                       | Comments                                   |
+|-------------------------------|------------|------------|-------------------------------|--------------------------------------------|
+| USER                          | yes        |            |                               | Username that EXTERNAL_HOST can ssh as     |
```

Comment:
What is the benefit of using USER or USERNAME instead of ATMOUSERNAME? We would have to define that variable somewhere. We could put this in defaults: `USER: "{{ATMOUSERNAME}}"`